knotx-server/src/main/java/io/knotx/server/configuration/KnotxServerConfiguration.java

```diff
@@ -23,6 +23,11 @@
 
 public class KnotxServerConfiguration {
 
+  //Default limit=-1 - Unlimited size
+  private final static long DEFAULT_UPLOAD_LIMIT = -1;
+
+  private final Long fileUploadLimit;
```

Comment:
Docs should be updated about new parameter.

Reply:
fixed